rsmtool/rsmsummarize.py

```diff
@@ -75,7 +77,15 @@ def check_experiment_dir(experiment_dir, configpath):
                                     "the .json configuration files for rsmtool "
                                     "experiments.".format(full_path_experiment_dir))
 
-        return jsons
+        # Raise an error if the user specified a list of experiment names 
+        # but we found several .jsons in the same directory
+        if experiment_name and len(jsons) > 1:
+            raise ValueError("{} contains several experiment jsons "
+                             "To use custom experiment names "
+                             "You must have a separate "
+                             "directory for each experiment.")
+        
+        return list(zip(jsons, [experiment_name]*len(jsons)))
```

Comment:
At this point, `jsons` _has to_ be a list of one element right? Couldn't add a comment to that effect and rewrite this as:

```suggestion
        return [jsons[0], experiment_name]
```

Or am I missing something? I guess I don't know what happens in the case where `experiment_name` evaluates to `False` and there are multiple JSONs in the experiment directory.

Reply:
If you have a single experiment in a folder and did not ask for an experiment name (default behavior), this would give you `[(exp_id, None)]`. If you have multiple experiments in the same folder (undocumented option), experiment name will have to be `None` and this will give you `[(exp1, None), (exp2, None), (exp3, None)]`. 

I added a comment to explain this.